certified-connectors/Impexium/apiDefinition.swagger.json

```diff
@@ -724,7 +724,10 @@
           "200": {
             "description": "default",
             "schema": {
-              "$ref": "#/definitions/CustomFieldData"
+              "type":  "array",
```

Comment:
@RjImpx The type has been changed for the output field. Existing flows may be broken if they are using this field and actual payload has been changed. May I ask you please to confirm that no users will be affected by this change?

Reply:
@Amjed-Ayoub confirming that users will not affected. We are fixing this bug.

Reply:
@RjImpx Thank you so much for the confirmation.